docs-dev/project/template/v.x.x.x/v.x.x.x-codename.md

```diff
@@ -0,0 +1,83 @@
+
```

Comment:
we should have only one template folder (guides & project are duplications)

also as mention before - incorporate task manager within the workflow